### PR TITLE
Announce 2.4.3

### PR DESCRIPTION
**Pull request by ktoso**
2 4 3!

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -5,16 +5,42 @@ title: Akka Documentation
...
+* `"com.typesafe.akka" % "akka-contrib_2.11"            % "2.4.3"`
+* `"com.typesafe.akka" % "akka-multi-node-testkit_2.11" % "2.4.3"`
+* `"com.typesafe.akka" % "akka-osgi_2.11"               % "2.4.3"`
+* `"com.typesafe.akka" % "akka-persistence_2.11"        % "2.4.3"`
+* `"com.typesafe.akka" % "akka-persistence-tck_2.11"    % "2.4.3"`
+* `"com.typesafe.akka" % "akka-remote_2.11"             % "2.4.3"`
+* `"com.typesafe.akka" % "akka-slf4j_2.11"              % "2.4.3"`
+* `"com.typesafe.akka" % "akka-stream_2.11"             % "2.4.3"`
+* `"com.typesafe.akka" % "akka-testkit_2.11"            % "2.4.3"`
+* all artifacts are also available for Scala `2.12.0-M3` as early preview
```
Comment (ktoso) [inline docs/index.md]: will replace this with fancy box soon

**Comment by patriknw** [comment]
LGTM

File: _includes/download.html [hunk 2]
```diff
@@ -104,11 +104,11 @@
             <div class="span6">
 
                 <div class="latest">
-                    <h2>Latest official version <small>Feb 16, 2016</small></h2>
+                    <h2>Latest official version <small>Apr 1, 2016</small></h2>
```
Comment (drewhk) [inline _includes/download.html]: oh no, it is not a joke!

File: docs/index.md [hunk 3]
```diff
@@ -5,16 +5,42 @@ title: Akka Documentation
...
+* `"com.typesafe.akka" % "akka-remote_2.11"             % "2.4.3"`
+* `"com.typesafe.akka" % "akka-slf4j_2.11"              % "2.4.3"`
+* `"com.typesafe.akka" % "akka-stream_2.11"             % "2.4.3"`
+* `"com.typesafe.akka" % "akka-testkit_2.11"            % "2.4.3"`
+* all artifacts are also available for Scala `2.12.0-M3` as early preview 
+
+As well as experimental modules:
+
+* `"com.typesafe.akka" % "akka-http-testkit-experimental_2.11"      % "2.4.3"`
+* `"com.typesafe.akka" % "akka-stream-experimental_2.11"            % "2.4.3"`
```
Comment (drewhk) [inline docs/index.md]: nope

**Comment by drewhk** [comment]
Apart from there is no `stream-experimental` for 2.4.3 looks great!

**Comment by ktoso** [comment]
Thanks for noticing the typo @drewhk

**Comment by ktoso** [comment]
I added a commit which makes this a reality:

akka_documentation___akka

It's not super hot, but it works :smirk:

**Comment by ktoso** [comment]
Resolves #300

**Pull request merged**